docs/servo_power_test/src/power_test_controller.ts

```diff
@@ -61,19 +54,27 @@ export class PowerTestController {
     await this.dutController.dut.close();
   }
   public async startMeasurement() {
-    if (this.ui.runnerNumber === 0) return;
+    if (this.ui.runnerNumber === 0) {
+      this.ui.setErrorMessage('No configuration/workload found.');
+      return;
+    }
     this.marginTime = Number(this.ui.marginTimeInput.value);
     this.iterationNumber = parseInt(this.ui.iterationInput.value);
-    if (this.iterationNumber <= 0) return;
-    await this.servoController.servoShell.select();
-    await this.dutController.dut.select();
-    await this.initialize();
+    if (this.iterationNumber <= 0) {
+      this.ui.setErrorMessage('Number of iterations <= 0.');
```

Comment:
ditto (error messages should include solutions, not only the cause of the error)